docs/usermanual/userman_admin.rst

```diff
@@ -60,7 +60,7 @@ Copy and paste the URL into a web browser. The KA Lite application should show u
     :focus: #id_username | Enter your username and password using this form!
     :class: screenshot
 
-* If you have forgotten the username/password combination, simply run ``bin/kalite manage createsuperuser`` from the kalite directory and then follow the instructions. If it is critical that you are able to login with your credentials, :doc:`../contact` and we can help you manually reset your login information.
+* If you have forgotten the username/password combination, simply run ``kalite manage createsuperuser`` from the kalite directory and then follow the instructions. If it is critical that you are able to login with your credentials, :doc:`../contact` and we can help you manually reset your login information.
```

Comment:
I just removed the sentence containing "contact". I think `kalite manage createsuperuser` should suffice for most needs.